src/styles/variables.scss

```diff
@@ -0,0 +1,15 @@
+$font-family: Arial;
+$day-size: 100px;
+$gap: 1px;
+$padding: 10px;
+$day-font-size: 30px;
+$day-color: #eee;
+$days: (
+  sun: 0,
+  mon: 1,
+  tue: 2,
+  wed: 3,
+  thu: 4,
+  fri: 5,
+  sut: 6,
```

Comment:
There appears to be a typo in the variable name for 'sut'. It should be 'sat' if it represents Saturday.